packages/tsconfig/base.json

```diff
@@ -5,7 +5,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
-    "lib": ["es2016", "es2017"],
+    "lib": ["es2019"],
```

Comment:
@jquense so technically node 10 doesn't have full es2019 support. but 12 is the current LTS and we should upgrade. and es(n) includes all the es(n-1)